bin/cassandra.in.sh

```diff
@@ -153,7 +153,9 @@ esac
 
 # Read user-defined JVM options from jvm-server.options file
 JVM_OPTS_FILE=$CASSANDRA_CONF/jvm${jvmoptions_variant:--clients}.options
-if [ $JAVA_VERSION -ge 17 ] ; then
```

Comment:
I'd go with -eq / ==, I don't think we support those non-LTS releases

Reply:
We do not support, but different people test different versions.
I think it will be most clear when we get from Apache trunk [CASSANDRA-18688](https://issues.apache.org/jira/browse/CASSANDRA-18688) and start using `CASSANDRA_JDK_UNSUPPORTED`
Let's leave it for now, please

Reply:
yes ge please.  Otherwise it is a pain to try out other versions.